ansible-ipi-install/roles/shared-labs-prep/files/99_sctp-module.yaml

```diff
@@ -0,0 +1,21 @@
+kind: MachineConfig
+metadata:
+  name: load-sctp-module
+  labels:
+    machineconfiguration.openshift.io/role: worker
+spec:
+  config:
+    ignition:
+      version: 3.1.0
```

Comment:
This ignition version will change based on OCP version. Can you ensure that this works for 4.5 ad above?

Reply:
Done @smalleni